packages/language-php/grammars/tree-sitter/queries/highlights.scm

```diff
@@ -245,22 +245,54 @@
 (class_declaration (base_clause (name) @entity.other.inherited-class.php))
 (class_declaration (class_interface_clause (name) @entity.other.implemented-interface.php))
 
+; usage or consumption of traits
+(use_declaration (name) @entity.name.type.trait.php)
+; in use lists, "bar" in `Foo::bar` is a method, not a constant
+(use_list
+  (_
+    (class_constant_access_expression (name) @support.other.function.method.php .)
+    (name) @support.other.function.method.php
+  )
+  (#set! capture.final true)
+)
+
+; the "Foo" and "bar" in `Foo::bar`
+(class_constant_access_expression . (name) @support.class.php)
+(class_constant_access_expression (name) @variable.other.property.php .)
```

Comment:
OK, I suppose we can't know whether or not `Suit::Spades` is an enum constant, so we'll have to treat it more generically as a class property.

Reply:
I think this makes sense. In use, `Suit::Spades` is treated more or less the same as a class constant, and I don't see how the parser can tell them apart, so scoping it as a class constant would make sense to me.